### PR TITLE
Update error-codes.md

### PR DESCRIPTION
**Pull request by avdvir**
add MI error code

### CODE REVIEW

**Comment by PRMerger4** [comment]
@avdvir : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by Jak-MS** [comment]
#label:"aq-pr-triaged"

File: data-explorer/error-codes.md [hunk 1]
```diff
@@ -54,6 +54,7 @@ The following list contains error codes you may come across during [ingestion](i
 | BadRequest_IngestionPropertyNotSupportedInThisContext | Ingestion property isn't supported in this context.| Permanent |
 | BadRequest_InvalidBlobUri                        | Blob URI is invalid.      | Permanent          |
 | BadRequest_DataCapacityLimitReached              | Storage account has reached its data capacity limit.      | Permanent          |
+| BadRequest_InvalidManagedIdentity                | Managed Identity either doesn't exist or doesn't has the right permissions.      | Permanent          |
```
Comment (shsagir) [inline data-explorer/error-codes.md:57]:
```suggestion
| BadRequest_InvalidManagedIdentity                | Managed Identity either doesn't exist or doesn't have the required permissions.      | Permanent          |
```

**Comment by shsagir** [comment]
@avdvir  Thanks for your suggestion. I approved it for publishing. In future, as an MSFT please use the private repo to create PRs. You can use the [SpineEdit](https://chrome.google.com/webstore/detail/spineedit/llhlgkbkfdfcbjbfnnakfpgmemopbbnf) browser extension to take you straight there from the page. Thx

#sign-off

**Comment by avdvir** [comment]
sign-off

**Comment by avdvir** [comment]
#sign-off

**Comment by PRMerger4** [comment]
@avdvir: I'm sorry - only the author of this article, @shsagir, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Pull request merged**